Samples/TrendingMovies/TrendingMovies-Benchmarking-UITests/TrendingMovies_Benchmarking_UITests.swift

```diff
@@ -0,0 +1,68 @@
+import XCTest
+
+class TrendingMovies_Benchmarking_UITests: XCTestCase {
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+    func testBenchmarkingOnScrolling() throws {
+        var avgPercentIncrease = 0.0
+        for _ in 0..<5 {
```

Comment:
I think most benchmarks usually run their code around ten times. Some even do some kind of warmup before the actual test. So our goal is to be under 5% on average even if there was no warm-up? How did you end up choosing five times? I think we should document the answers to my questions either here as a code comment or somewhere else.

Reply:
Noted in our planning doc and will update #1834